i915.c

```diff
@@ -964,6 +978,10 @@ static int i915_bo_compute_metadata(struct bo *bo, uint32_t width, uint32_t heig
 		bo->meta.tiling = I915_TILING_4;
 		break;
 	}
+	char *pFormat = (char*)&format;
+	drv_loge("-zcy- %s() w/h=%d/%d format=%c%c%c%c use_flags=0x%x mod/cnt=0x%x/%d tiling=0x%x\n", __func__,
```

Comment:
please remove the -zcy

Reply:
For debug purpose, will remove the log when test is ok